p2p/conn/connection.go

```diff
@@ -249,9 +255,6 @@ func (c *MConnection) FlushStop() {
 		// so we dont race on calling sendSomePacketMsgs
 		<-c.doneSendRoutine
 
-		// Send and flush all pending msgs.
```

Comment:
why remove `// Send and flush all pending msgs.`? looks like we're still flushing the messages

Reply:
Yea - I will put this back. The `IsRunning` comment is the part that was wrong. I accidentally removed this when I should not have.

Reply:
Ok, thanks